src/CTA.Rules.Update/CodeReplacer.cs

```diff
@@ -83,10 +89,50 @@ public CodeReplacer(List<SourceFileBuildResult> sourceFileBuildResults, ProjectC
                 {
                     LogHelper.LogError(new FilePortingException(Constants.Project, new Exception("Error adding project to actions collection")));
                 }
+
+                if (_projectConfiguration.ProjectType == ProjectType.WCFConfigBasedService || _projectConfiguration.ProjectType == ProjectType.WCFCodeBasedService)
+                {
+                    RunWCFChanges();
+                }
             }
             return actionsPerProject.ToDictionary(a => a.Key, a => a.Value);
         }
 
+        private void RunWCFChanges()
```

Comment:
As discussed before, please add an issue to move this to an action later so that it's consistent with the way we run actions.